config/crd/bases/addon.open-cluster-management.io_managedclusteraddons.yaml

```diff
@@ -0,0 +1,256 @@
+
```

Comment:
this file shouldn't be here.

Reply:
had to add it because of (same for ManagedCluster CRD)
`no matches for kind "ManagedClusterAddOn" in version "addon.open-cluster-management.io/v1alpha1" occurred` error. Should I keep these files when testing in local and remove them while pushing the PR ??

Reply:
Keep these files in test directory.